webmails/roundcube/Dockerfile

```diff
@@ -1,72 +1,56 @@
- && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
- && rm -rf /var/lib/apt/lists
+RUN apk add --no-cache \
+    nginx gpg gpg-agent \
+    php8 php8-fpm php8-mbstring php8-zip php8-xml php8-simplexml \
+    php8-dom php8-curl php8-exif gd php8-gd php8-iconv php8-intl php8-openssl \
+    php8-pdo_sqlite php8-pdo_mysql php8-pdo_pgsql php8-pdo php8-sodium libsodium php8-tidy php8-pecl-uuid \
+    php8-pspell php8-pecl-imagick php8-opcache php8-session php8-sockets php8-fileinfo \
+    && rm /etc/nginx/http.d/default.conf \
+    && rm /etc/php8/php-fpm.d/www.conf \
+    && mkdir -p /run/nginx \
+    && mkdir -p /conf
 
 ENV ROUNDCUBE_URL https://github.com/roundcube/roundcubemail/releases/download/1.5.3/roundcubemail-1.5.3-complete.tar.gz
-ENV CARDDAV_URL https://github.com/mstilkerich/rcmcarddav/releases/download/v4.4.2/carddav-v4.4.2.tar.gz
+ENV CARDDAV_URL https://github.com/mstilkerich/rcmcarddav/releases/download/v4.4.3/carddav-v4.4.3.tar.gz
 
-RUN set -eu \
- && rm -rf /var/www/html/ \
- && cd /var/www \
+RUN \
+ cd /var/www \
  && curl -sL ${ROUNDCUBE_URL} | tar xz \
  && curl -sL ${CARDDAV_URL} | tar xz \
- && mv roundcubemail-* html \
- && mv carddav html/plugins/ \
- && cd html \
+ && mv roundcubemail-* webmail \
+ && mkdir -p /var/www/webmail/config \
+ && mv carddav webmail/plugins/ \
+ && cd webmail \
  && rm -rf CHANGELOG.md SECURITY.md INSTALL LICENSE README.md UPGRADING composer.json-dist installer composer.* \
- && ln -sf index.php /var/www/html/sso.php \
- && ln -sf /dev/stderr /var/www/html/logs/errors.log \
- && chown -R root:root . \
- && chown www-data:www-data logs temp \
- && chmod -R a+rX . \
- && a2enmod rewrite deflate expires headers \
- && echo date.timezone=${TZ} > /usr/local/etc/php/conf.d/timezone.ini \
+ && ln -sf index.php /var/www/webmail/sso.php \
+ && chmod -R u+w,a+rX /var/www/webmail \
+ && chown -R nginx:nginx /var/www/webmail \
  && rm -rf plugins/{autologon,example_addressbook,http_authentication,krb_authentication,new_user_identity,password,redundant_attachments,squirrelmail_usercopy,userinfo,virtuser_file,virtuser_query}
 
-# enable database_attachments (and memcache?)
 
-COPY mailu.php /var/www/html/plugins/mailu/mailu.php
-COPY php.ini /
-COPY config.inc.php /
+# nginx / PHP config files
+COPY config/nginx-roundcube.conf /conf/nginx-roundcube.conf
+COPY config/php-roundcube.conf /etc/php8/php-fpm.d/roundcube.conf
+COPY config/php.ini /conf/php.ini
+COPY config/config.inc.php /conf/config.inc.php
+COPY login/mailu.php /var/www/webmail/plugins/mailu/mailu.php
+COPY config/config.inc.carddav.php /var/www/webmail/plugins/carddav/config.inc.php
+
 COPY start.py /
-COPY config.inc.carddav.php /var/www/html/plugins/carddav/config.inc.php
 
 EXPOSE 80/tcp
 VOLUME /data
 VOLUME /overrides
 
 CMD /start.py
 
-HEALTHCHECK CMD curl -f -L -H 'User-Agent: health' http://localhost/ || exit 1
+HEALTHCHECK CMD curl -f -L http://localhost/ping || exit 1
```

Comment:
See comment about IPv6 below

Reply:
You removed the User-Agent - do you still filter the checks from the access log?
This can be done in the nginx /ping location

Reply:
the webmails have the access log disabled.